articles/iot-edge/tutorial-deploy-machine-learning.md

```diff
@@ -13,7 +13,7 @@ ms.custom: mvc
 
 # Deploy Azure Machine Learning as an IoT Edge module - preview
 
-You can use IoT Edge modules to deploy code that implements your business logic directly to your IoT Edge devices. This tutorial walks you through deploying an Azure Machine Learning module that predicts when a device fails based on simulated machine temperature data. 
+You can use IoT Edge modules to deploy code that implements your business logic directly to your IoT Edge devices. This tutorial walks you through deploying an Azure Machine Learning module that predicts when a device fails based on simulated machine temperature data. For more information about Azure ML on IoT Edge, see [Azure Machine Learning documentation](https://docs.microsoft.com/en-us/azure/machine-learning/desktop-workbench/use-azure-iot-edge-ai-toolkit).
```

Comment:
Best practice is to use relative paths to other articles in the repo. So this would be `../machine-learning/desktop-workbench/use-azure-iot-edge-ai-toolkit.md`. 

When you do have to use absolute URLs, it's important to remove any locale-tags. In this case, `/en-us`. That way, people in other countries will see the localized versions based on the language they've chosen in their browser. You don't want to hard-code a language into a link. Doesn't apply here, but good to know.